core/unit_test/tools/TestWithoutInitializing.cpp

```diff
@@ -75,3 +75,38 @@ TEST(kokkosp, create_mirror_no_init) {
       });
   ASSERT_TRUE(success);
 }
+
+TEST(kokkosp, create_mirror_no_init_view_ctor) {
+  using namespace Kokkos::Test::Tools;
+  listen_tool_events(Config::DisableAll(), Config::EnableKernels());
+  Kokkos::View<int*, Kokkos::DefaultExecutionSpace> device_view("device view",
+                                                                10);
+  Kokkos::View<int*, Kokkos::HostSpace> host_view("host view", 10);
+
+  auto success = validate_absence(
+      [&]() {
+        auto mirror_device = Kokkos::create_mirror(
+            Kokkos::view_alloc(Kokkos::HostSpace{},
+                               Kokkos::WithoutInitializing),
+            device_view);
+        auto mirror_host = Kokkos::create_mirror(
+            Kokkos::view_alloc(Kokkos::HostSpace{}, Kokkos::WithoutInitializing,
+                               Kokkos::DefaultExecutionSpace{}),
+            host_view);
+        auto mirror_device_view = Kokkos::create_mirror_view(
+            Kokkos::view_alloc(Kokkos::HostSpace{},
+                               Kokkos::WithoutInitializing),
+            device_view);
+        auto mirror_host_view = Kokkos::create_mirror_view(
+            Kokkos::view_alloc(Kokkos::HostSpace{}, Kokkos::WithoutInitializing,
+                               Kokkos::DefaultExecutionSpace{}),
+            host_view);
```

Comment:
can we test the `view_alloc(WithoutInitializing)` too?

Reply:
Sure, see 8c9aefb8bba6f656cedaafc3232a4fd1974fa638.